src/app/static/src/app/components/files/ManageFile.vue

```diff
@@ -60,6 +61,7 @@
         valid: boolean,
         error: Error,
         existingFileName: string,
+        hasValidAdr: boolean,
```

Comment:
this prop name doesn't match the prop you're passing to it here - https://github.com/mrc-ide/hint/pull/387/files#diff-ee212c213967126a949944618cc04cbb58942c8165a8ed85b07790a4f4056c74R30 

as a result it will never be true